CMakeLists.txt

```diff
@@ -242,6 +242,15 @@ Halide_feature(WITH_TESTS "Halide's unit test suite" TOP_LEVEL)
 Halide_feature(WITH_TUTORIALS "Halide's tutorial code" TOP_LEVEL)
 Halide_feature(WITH_UTILS "Optional utility programs for Halide, including HalideTraceViz" TOP_LEVEL)
 
+##
+# FIXME: Disable other GPU backends 
+if (Halide_TARGET MATCHES "vulkan")
+    message(WARNING "Building Vulkan runtime support!")
+else ()
+    message(FATAL_ERROR "Target disabled for testing ...")
```

Comment:
This aborts CMake configuration. Is that on purpose??

Reply:
Yes, that was the intent.  I was trying to force the buildbot to skip all other targets since it takes ~2hr before the Vulkan build even starts.  Unfortunately, if one of the configure steps fails, the buildbot script won't continue and fails the entire build.  So I reverted the above.